code/modules/hydroponics/plant_genes.dm

```diff
@@ -485,6 +485,19 @@
 			HY.update_icon()
 			HY.visible_message("<span class='warning'>The [H.myseed.plantname] spreads!</span>")
 
+/datum/plant_gene/trait/brewing
+	name = "Auto-Distilling Composition"
+
+/datum/plant_gene/trait/eyes
+	name = "Oculary Mimicry"
+	var/mutable_appearance/googly
```

Comment:
Please add [dmdoc documentation](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md) to be documentation compliant